crates/cxx-qt-build/src/lib.rs

```diff
@@ -491,13 +491,25 @@ impl CxxQtBuilder {
             "cargo:rustc-cfg=cxxqt_qt_version_major=\"{}\"",
             qtbuild.version().major
         );
+
         // Also set to env so our cfg evaulator finds it
         // note we need this before generate_cxxqt_cpp_files
         env::set_var(
             "CARGO_CFG_cxxqt_qt_version_major",
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version_{}_{}",
+                qtbuild.version().major,
+                minor
+            );
+            println!("cargo:rustc-cfg={}", at_least_qt_version);
```

Comment:
wonder if this should be set to true as well so `println!("cargo:rustc-cfg={}=\"true\"", at_least_qt_version);` or maybe that would then mean that code outside of CXX bridges would require `#[cfg(cxxqt_qt_least_qt_version_6_5 = "true")]` rather than `#[cfg(cxxqt_qt_least_qt_version_6_5)]`.

Can you see what happens at the moment for a method outside of the CXX bridges (eg normal Rust code) does `#[cfg(cxxqt_qt_least_qt_version_6_5)]` work with the existing code?

Reply:
```
  --> examples/qml_features/rust/src/invokables.rs:62:12
   |
10 |   #[cxx_qt::bridge(cxx_file_stem = "rust_invokables")]
   |   ---------------------------------------------------- function or associated item `foo` not found for this struct
...
21 |           type RustInvokables = super::RustInvokablesRust;
   |  ______________-
22 | |     }
23 | |
24 | |     // ANCHOR: book_qnamespace
...  |
61 | |         #[cfg(cxxqt_qt_least_qt_version_6_9)]
62 | |         fn foo(self: Pin<&mut RustInvokables>);
   | |           -^^^ function or associated item not found in `RustInvokables`
   | |___________|
   | 
```
=> doesn't work.

Reply:
Isn't this inside a CXX / CXX-Qt bridge macro, i was meaning outside the bridge, as that is where `cargo:rustc-cfg` end up

Reply:
```
diff --git a/crates/cxx-qt-build/src/lib.rs b/crates/cxx-qt-build/src/lib.rs
index 73be1ae0..10bbf51b 100644
--- a/crates/cxx-qt-build/src/lib.rs
+++ b/crates/cxx-qt-build/src/lib.rs
@@ -504,7 +504,7 @@ impl CxxQtBuilder {
                 qtbuild.version().major,
                 minor
             );
-            println!("cargo:rustc-cfg={}", at_least_qt_version);
+            println!("cargo:rustc-cfg={}=\true\"", at_least_qt_version);
             let variable_cargo = format!("CARGO_CFG_{}", at_least_qt_version);
             env::set_var(variable_cargo, "true");
         }

```

compile failed
```
   Compiling qml-features v0.1.0 (/opt/travail/git/cxx-qt/examples/qml_features/rust)
error[E0765]: unterminated double quote string
  |
  = note: this error occurred on the command line: `--cfg=cxxqt_at_least_qt_version_6_0=    rue"`

error: could not compile `cxx-qt-lib` (lib)
```

Reply:
OK tested locally this doesn't need the true for the `cargo:rustc-cfg` so what you have currently is fine

Reply:
Think you had a typo `cxxqt_qt_least_qt_version_6_5` should be `cxxqt_at_least_qt_version_6_5` :-)

Reply:
ok